lib/command.js

```diff
@@ -108,12 +108,12 @@ module.exports = function (yargs, usage, validation) {
       if (/\.+[\]>]/.test(cmd) && i === splitCommand.length - 1) variadic = true
       if (/^\[/.test(cmd)) {
         parsedCommand.optional.push({
-          cmd: cmd.replace(bregex, ''),
+          cmd: cmd.replace(bregex, '').split('|'),
```

Comment:
Would be cool to allow whitespace between the aliases, e.g.

```js
yargs.command('get <username | email> [ password ]', 'fetch a user by username or email.')
```

We can make this happen with the following 4 changes:

1. Change line 99 to this:

    ```js
    var splitCommand = extraSpacesStrippedCommand.split(/\s+(?![^\[]*\]|[^<]*>)/)
    ```

2. Change line 108 to this:

    ```js
    if (/\.+[\]>]/.test(cmd.replace(/\s/g, '')) && i === splitCommand.length - 1) variadic = true
    ```

3. Change line 111 to this:

    ```js
    cmd: cmd.replace(/\s/g, '').replace(bregex, '').split('|'),
    ```

4. Change line 116 to this:

    ```js
    cmd: cmd.replace(/\s/g, '').replace(bregex, '').split('|'),
    ```

All tests still pass this way. Let me know what you think.